### ISSUE TITLE
Here string intellisense completion and indentation

### ISSUE THREAD

**Issue by DarkLite1** [PowerShell/vscode-powershell#1517]
In the editor here strings are not correctly completed and indented. some expamples:

![image](https://user-images.githubusercontent.com/10828570/45204817-ae9ec980-b280-11e8-92b5-20c28453a6e1.png)

This is very annoying when working with HTML strings.

```powershell
$pseditor.EditorServicesVersion
Major  Minor  Build  Revision
-----  -----  -----  --------
1      8      4      0

Version: 1.28.0-insider (user setup)
Commit: a3f9f2e7240106aeec2f1d4bc03d7876bd42e2e4
Date: 2018-09-07T05:20:29.346Z
Electron: 2.0.7
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64
```

**Comment by tylerl0706** [comment]
Yep. This is a duplicate of https://github.com/PowerShell/vscode-powershell/issues/181. We just haven't had cycles to address the problem.

I would think one would just need to send a PR to VSCode itself and add to this
https://github.com/Microsoft/vscode/blob/master/extensions/powershell/language-configuration.json#L11

But I haven't tried it.

**Comment by DarkLite1** [comment]
Maybe @aeschli can have a look? It would be great if this could be fixed.

**Comment by tylerl0706** [comment]
btw anyone should be able to play around with this. The file should be located here:
```
C:\Program Files\Microsoft VS Code\resources\app\extensions\powershell\language-configuration.json
```
(I'm on a mac so it's slightly different `/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/powershell/language-configuration.json`)

Try adding `["@'", "'@"]` - and if that doesn't work, as a test, remove the `{ "open": "'", "close": "'", "notIn": ["string", "comment"]},`

If that works, we should get some guidance from @aeschli on how we can have both of these entries in the `langauge-configuration.json`

**Comment by DarkLite1** [comment]
Thanks for the help @tylerl0706 . I've tried your suggestion:

C:\Users\bgijbels\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\extensions\powershell\language-configuration.json

```json
{
	"comments": {
		"lineComment": "#",
		"blockComment": [ "<#", "#>" ]
	},
	"brackets": [
		["{", "}"],
		["[", "]"],
		["(", ")"],
		["@'", "'@"]
	],
	"autoClosingPairs": [
		["{", "}"],
		["[", "]"],
		["(", ")"],
		{ "open": "\"", "close": "\"", "notIn": ["string"]},
		["<#", "#>"],
		["@'", "'@"]
	],
	"surroundingPairs": [
		["{", "}"],
		["[", "]"],
		["(", ")"],
		["\"", "\""],
		["'", "'"],
		["@'", "'@"]
	],
	"folding": {
		"markers": {
			"start": "^\\s*#[rR]egion\\b",
			"end": "^\\s*#[eE]nd[rR]egion\\b"
		}
	}
}
```

Nothing changed. After restarting VS Code Insiders it even threw an error that the PowerShell extension crashed. But that hasn't happened anymore after the first time.

**Comment by rkeithhill** [comment]
Try modifying just this section:
```
    "autoClosingPairs": [
        ["{", "}"],
        ["[", "]"],
        ["(", ")"],
        ["@'", "\n'@"],
        ["@\"", "\n\"@"],
        ["<#", "#>"],
    ],
```
That works for me but now you lose auto-close of `""` and `''`.  :-(

**Comment by tylerl0706** [comment]
Yep. I get the same thing as @rkeithhill.

So I think we should open an issue on vscode for guidance.

**Comment by tylerl0706** [comment]
Opened this https://github.com/Microsoft/vscode/issues/58619

**Comment by tylerl0706** [comment]
@rkeithhill I agree 100%. 😄

**Comment by DarkLite1** [comment]
Thanks @tylerl0706 and @rkeithhill for putting it in the spotlight, been struggling with this for a while now. If this gets fixed than VS Code will be my main editor.
